docs/zkapps/tutorials/05-common-types-and-functions.mdx

```diff
@@ -2,7 +2,7 @@
 title: 'Tutorial 5: Common Types and Functions'
 hide_title: true
 sidebar_label: 'Tutorial 5: Common Types and Functions'
-description: Higher-order types are built from Fields and are useful for your development. Learn how to create custom types.
+description: Higher-order types are built from Fields. Create custom types to build your own strings. Create compound data types with the Struct.
```

Comment:
> Create custom types to build your own strings

I think we should slightly change this, since this tutorial isn't only about strings, its more about common data types, functions and how to compose and use them